src/main/resources/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewAction/index.jelly

```diff
@@ -5,7 +5,7 @@
          xmlns:t="/lib/hudson">
   <l:layout title="${%Graph} [${it.fullBuildDisplayName}]" type="one-column">
     <l:main-panel>
-      <link rel="stylesheet" href="${resURL}/plugin/pipeline-graph-view/js/style.css" type="text/css" />
+      <link rel="stylesheet" href="${rootURL}/plugin/pipeline-graph-view/js/style.css" type="text/css" />
```

Comment:
why change from resURL to rootURL?

resURL ensures that caching is handled appropriately by inserting characters in the path that change as needed.